reporters/GlobalSummaryReporter.js

```diff
@@ -325,28 +324,20 @@ class GlobalSummaryReporter {
 
     // - GCR (collateral / tokens outstanding):
     let currentCollateral = this.toBN(this.totalPositionCollateral.toString());
-    let prevPeriodCollateral = currentCollateral.sub(netCollateralWithdrawnPeriod);
     let currentTokensOutstanding = this.toBN(this.totalTokensOutstanding.toString());
-    let prevPeriodTokensOutstanding = currentTokensOutstanding.sub(netTokensMintedPeriod);
     let currentGCR = currentCollateral.mul(this.toBN(this.toWei("1"))).div(currentTokensOutstanding);
-    let prevGCR = prevPeriodCollateral.mul(this.toBN(this.toWei("1"))).div(prevPeriodTokensOutstanding);
     allSponsorStatsTable["GCR - collateral / # tokens outstanding"] = {
-      current: this.formatDecimalString(currentGCR),
```

Comment:
I decided to remove the GCR and reference price delta data because there is no data being counted for these stats for the "period". We only report the current price, therefore the delta stat wouldn't be correct here if the "period-offset > 0" (i.e. the period does not end at the current time)